weblate/static/style-bootstrap.css

```diff
@@ -1617,10 +1617,18 @@ input.color_edit:checked + label {
   color: #2a3744;
 }
 
+table:has(#machinery-translations) {
+  table-layout: fixed;
+}
+
 .machinery-text {
   word-wrap: anywhere;
 }
 
+.machinery-diff {
+  word-wrap: break-word;
```

Comment:
changing this CSS rule to `word-break: break-word` would even work on it's own

Reply:
Maybe it would be a better approach to use just this? I'm afraid that with a fixed layout, it's impossible to account for all possible situations (for example, buttons have varying length in different translations).